meinberlin/apps/bplan/tasks.py

```diff
@@ -19,59 +19,40 @@ def get_features_from_bplan_api(endpoint):
     return res_json.get("features")
 
 
-def get_bplan_point_and_district_pk(bplan_identifier):
+def get_bplan_point(bplan_identifier):
     url_poi = "bplan/points/" + "?bplan={}".format(bplan_identifier.replace(" ", "%20"))
 
     try:
         features = get_features_from_bplan_api(url_poi)
         if features:
-            district_pk = features[0]["properties"]["bezirk"]
             point = features[0]
+            return point
 
-            return point, district_pk
-
-        return None, None
+        return None
 
     except UnicodeEncodeError:
         # catches bplan-identifiers with problematic chars
-        return None, None
-
-
-def get_bplan_api_pk_to_a4_admin_district_dict():
-    url_dis = "bezirke/"
-    features = get_features_from_bplan_api(url_dis)
-    dis_dict = {}
-    if features:
-        for district in features:
-            dis_model = AdministrativeDistrict.objects.filter(
-                name=district["properties"]["name"]
-            )
-            if dis_model:
-                dis_dict[district["properties"]["pk"]] = dis_model[0]
-            else:
-                dis_dict[district["properties"]["pk"]] = None
-
-    return dis_dict
+        return None
 
 
 @shared_task
 def get_location_information(bplan_id: int):
     """Fetch and update the location information for a bplan.
     Called via the post_save signal in signals.py. To prevent the signal
     from retriggering after updating the location information save() needs
-    to be called with the update_fields parameter: save(update_fields=["point", "admistrative_district"])
+    to be called with the update_fields parameter: save(update_fields=["point", "admistrative_district"]).
```

Comment:
not sure why in the string docs you keep the `administrative_district` while you are saving it in the end without it.

Reply:
forgot to remove it, thanks